lms/static/sass/search/_search.scss

```diff
@@ -0,0 +1,174 @@
+.search-bar,
+.course-index .search-bar,
+.dashboard .search-bar {
+
+  @include box-sizing(border-box);
+  position: relative;
+  padding: 5px;
+  font-family: $sans-serif;
+
+  form > div {
+    position: relative;
+  }
+
+  .search-field {
+    @include box-sizing(border-box);
+    @include border-radius(4px);
+    top: 5px;
+    width: 100%;
+    background: $white-t1;
+    font-family: $sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    letter-spacing: normal;
```

Comment:
is letter spacing acting as a true override here? I'm seeing it a number of places, and I'm not sure if it is a necessary reset or could be dropped. If it isn't an override, I would remove this rule.